build/common.sh

```diff
@@ -67,15 +67,14 @@ readonly LOCAL_OUTPUT_IMAGE_STAGING="${LOCAL_OUTPUT_ROOT}/images"
 # This is a symlink to binaries for "this platform" (e.g. build tools).
 readonly THIS_PLATFORM_BIN="${LOCAL_OUTPUT_ROOT}/bin"
 
-readonly OUTPUT_BINPATH="${CUSTOM_OUTPUT_BINPATH:-$LOCAL_OUTPUT_BINPATH}"
```

Comment:
This allows for users to pick the binpath for really remote docker engines.  I'm not sure who is using it.  I sent out mail on this a while ago and @MHBauer (I think) still hits this case.